dali/dafilesrv/dafilesrv.cpp

```diff
@@ -691,6 +692,8 @@ int main(int argc,char **argv)
     } perfHook;
     startPerformanceMonitor(10*60*1000, PerfMonStandard, &perfHook);
     writeSentinelFile(sentinelFile);
+
+    setKeyIndexCacheSize(0); // disable IKeyIndex caching
```

Comment:
I think we need to revisit this separately.
In new cleaned up PR (about to open) I have a slightly better solution I think, where instead of disabling, I keep the cache and flush the specific key on client close.
That at least avoids the problem you do hit otherwise where a updated key (e.g. format) hits errors.

I do wonder if disabling the cache completely might be best in dafilesrv and therefore instead rely on the disk cache.
Unfortunately setKeyIndexCacheSize(0) does not really disable it completely, the jhtree key code is so heavily inter-twined with the node cache that it will still use it.
It is in need of some refactoring as we have discussed before for other reasons too.